apiserver/daemon.go

```diff
@@ -526,6 +526,18 @@ func (d *DaemonConfig) handleCopy(w http.ResponseWriter, r *http.Request) {
 		)).Combine(err))
 		return
 	}
+
+	content, err := json.Marshal(newVolConfig)
+	if err != nil {
+		api.RESTHTTPError(w, errors.PublishVolume.Combine(errored.Errorf(
```

Comment:
The overall operation is publish; you'll notice almost all the other errors use this constant as well.